tree/ntuple/v7/inc/ROOT/RColumnElement.hxx

```diff
@@ -248,36 +248,25 @@ namespace Detail {
 
 // clang-format off
 /**
-\class ROOT::Experimental::Detail::RColumnElement
+\class ROOT::Experimental::Detail::RColumnElementBase
 \ingroup NTuple
-\brief A column element points either to the content of an RFieldValue or into a memory mapped page.
+\brief A column element encapsulates the translation between basic C++ types and their column representation.
 
-The content pointed to by fRawContent can be a single element or the first element of an array.
 Usually the on-disk element should map bitwise to the in-memory element. Sometimes that's not the case
-though, for instance on big endian platforms and for exotic physical columns like 8 bit float.
+though, for instance on big endian platforms or for bools.
 
-This class does not provide protection around the raw pointer, fRawContent has to be managed correctly
-by the user of this class.
+There is a template specialization for every valid pair of C++ type and column representation.
+These specialized child classes are responsible for packing and unpacking pages.
```

Comment:
Given that `Pack()` or `Unpack()` do not take a `RPage` as argument, IMO, the suggestion below would be more precise.
```suggestion
These specialized child classes are responsible for overriding `Pack()` / `Unpack()` for packing / unpacking elements
as appropriate.
```